platforms/spring-boot/components-starter/camel-salesforce-starter/src/main/java/org/apache/camel/component/salesforce/springboot/SalesforceComponentConfiguration.java

```diff
@@ -832,6 +832,10 @@ public void setApprovalSkipEntryCriteria(
          */
         private String clientSecret;
         /**
+         * Salesforce connected application Consumer token
```

Comment:
To keep in in line could this be `Salesforce connected application OAuth refresh token`?